pdtf-api-1.2.0.yaml

```diff
@@ -0,0 +1,177 @@
+openapi: 3.0.3
+info:
+  title: Property Data Trust Framework API
+  version: 1.2.0
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  description: This API defines how participants in the UK Property Data Trust Framework can securely share property data, together with the verifiable provenance of that data. The API is built around a variation of OpenID Connect Verified Claims schema.
+servers:
+  - url: http://api.pdtf-example.com
+    description: Example PDTF data source
+tags:
+  - name: transactions
+    description: Transactions API
+paths:
+  /transactions:
+    post:
+      tags:
+        - transactions
+      summary: Create a transaction
+      operationId: addTransactionClaims
+      responses:
```

Comment:
The only thing I would comment Amol is that I think response should return the new transactionId (the transaction is only created if there are no errors in the claims)

I would possibly say in REST-ful terms this endpoint should only create the transaction, with no claims added, but I can see how it might be convenient for some systems to do both.

Reply:
@edmolyneux Updated to return the transaction ID in the `location` header